### PR TITLE
examples/flake.nix: minor improvements

### PR DESCRIPTION
**Pull request by erikarvstedt**
Add some improvements in response to a user question from Matrix.

### CODE REVIEW

**Comment by seberm** [comment]
Hello,
maybe a silly question but is there some difference when I want to deploy the flake system configuration using krops? How can I do that?

Thanks

**Comment by erikarvstedt** [comment]
Krops is not recommended. Use a modern deployment tool like colmena or morph. Details: https://github.com/fort-nix/nix-bitcoin/issues/556.

File: examples/flakes/flake.nix [hunk 1]
```diff
@@ -8,13 +8,14 @@
 {
   description = "A basic nix-bitcoin node";
 
-  inputs.nix-bitcoin.url = "github:fort-nix/nix-bitcoin/release";
-  # You can also use a version branch to track a specific NixOS release
-  # inputs.nix-bitcoin.url = "github:fort-nix/nix-bitcoin/nixos-23.05";
+  inputs.nix-bitcoin.url = "github:fort-nix/nix-bitcoin/nixos-23.05";
```
Comment (jonasnick) [inline examples/flakes/flake.nix:11]: Doesn't this mean that `nix flake update` will not actually update your system anymore when we update to a new NixOS release? That seems undesirable - at least as a default option. Is the user supposed to notice that there are not updates coming from the nix-bitcoin inputs anymore?

Comment (erikarvstedt) [reply examples/flakes/flake.nix]: Agreed. I've reverted this change.

**Review by jonasnick — APPROVED** [review]
ACK 742fd8fdd0a66d705def05e2e85522ed1561a813

**Pull request merged**